hosts/ficolobuild/builder.nix

```diff
@@ -45,6 +45,11 @@
   # Environment for Yubikey provisioning
   environment.systemPackages = with pkgs; [
     usbutils
+    screen
+    (python310.withPackages (ps:
+      with ps; [
+        requests
+      ]))
```

Comment:
Why do we need to install python 3.10 in systemPackages? What is the exact use-case here?

Reply:
What the other choices would be? Assuming that requests is needed

Reply:
The package that needs python 3.10 and specific python libraries should state its own dependencies, rather than have them in the systemPackages.

Reply:
This is for a hydra container setup, which is not properly packaged (and won't be)